docs/content/en/blog/news/february-2022-update.md

```diff
@@ -0,0 +1,97 @@
+---
+date: 2022-02-10
+title: "February 2022 update"
+linkTitle: "February 2022 update"
+weight: 996
+description: "Development status update to recap what happened in January"
+author: Le Van Nghia ([@nghialv](https://twitter.com/nghialv2607))
+---
+
+_Published by the PipeCD dev team every month, this update will provide you with news and updates about the project! Please click [here](/blog/2022/01/05/january-2022-update/) if you want to see the last status update._
+
+### What's changed
+---
+
+Since the last report, PipeCD team has introduced 4 releases ([v0.24.0](https://github.com/pipe-cd/pipecd/releases/tag/v0.24.0), [v0.24.5](https://github.com/pipe-cd/pipecd/releases/tag/v0.24.5), [v0.25.0](https://github.com/pipe-cd/pipecd/releases/tag/v0.25.0), [v0.25.1](https://github.com/pipe-cd/pipecd/releases/tag/v0.25.1)). This blog post walks you through their notable changes. For all other changes, please check out each release note to see.
+
+#### Introducing Label mechanism
+
+Label concept has been introduced. Labels are key/value pairs that are attached to PipeCD resources such as application, deployment... As a developer, you can use labels to add identifying attributes to them. We believe that supporting the Label will give us more flexibility on grouping, filtering as well as managing access control on PipeCD resources.
+
+Application labels can be specified via the `labels` field in the application configuration file while deployment labels are inherited from its application.
+
+```yaml
+apiVersion: pipecd.dev/v1beta1
+kind: KubernetesApp
+spec:
+  name: example
+  labels:
+    env: prod
+    team: product
+```
+
+From `v0.24.0`, application labels are shown on the application list and detail page. All filter forms also have a new label box to allow filtering resources by their labels.
+
+#### Environment and Deployment Configuration File were deprecated
+
+Since the Label was introduced as above, Environment was becoming a subset of the Label concept. So we decided to deprecate the Environment concept from `v0.24.0`. Although it is still being displayed on the web console, it will be removed completely from the next release. So make sure to switch to using Label instead.
```

Comment:
```suggestion
Since the Label was introduced, the Environment becomes a subset of the Label concept. So we decided to deprecate the Environment concept from `v0.24.0`. Even though it is still displayed on the web console, it will be removed completely from the next release. So make sure to switch to using Label instead.
```